ocaml/xapi/xapi_clustering.ml

```diff
@@ -522,11 +522,16 @@ module Watcher = struct
 
   let cluster_change_watcher : bool Atomic.t = Atomic.make false
 
+  (* this is the time it takes for the update request to time out. It is ok to set
+     it to a relatively long value since the call will return immediately if there
+     is an update *)
+  let cluster_change_interval = Mtime.Span.min
```

Comment:
note, min is for minute, not minimum

Reply:
👍 which I think I have used in the right way